microk8s-resources/actions/disable.dns.sh

```diff
@@ -6,7 +6,8 @@ set -e
 skip_opt_in_config() {
```

Comment:
Please add a brief comment explaining what this function does.

Reply:
Done